[rospy.client][INFO] 2021-09-09 15:30:40,496: init_node, name[/server_endpoint], pid[28448]
[xmlrpc][INFO] 2021-09-09 15:30:40,497: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-09-09 15:30:40,497: Started XML-RPC server [http://wng:34891/]
[rospy.impl.masterslave][INFO] 2021-09-09 15:30:40,497: _ready: http://wng:34891/
[rospy.init][INFO] 2021-09-09 15:30:40,497: ROS Slave URI: [http://wng:34891/]
[rospy.registration][INFO] 2021-09-09 15:30:40,497: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2021-09-09 15:30:40,497: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-09-09 15:30:40,597: registered with master
[rospy.rosout][INFO] 2021-09-09 15:30:40,598: initializing /rosout core topic
[rospy.rosout][INFO] 2021-09-09 15:30:40,601: connected to core topic /rosout
[rospy.simtime][INFO] 2021-09-09 15:30:40,603: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-09-09 15:30:40,617: Starting server on 192.168.1.146:10000
[rospy.internal][INFO] 2021-09-09 15:30:40,843: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2021-09-09 15:31:09,452: Connection from 192.168.1.146
[rosout][INFO] 2021-09-09 15:31:09,457: RegisterPublisher(execution_check, <class 'panda_move.msg._ExecutionCheck.ExecutionCheck'>) OK
[rospy.internal][INFO] 2021-09-09 15:31:09,538: topic[/execution_check] adding connection to [http://wng:34891/], count 0
[rospy.internal][INFO] 2021-09-09 15:31:09,539: topic[/execution_check] adding connection to [/server_endpoint], count 0
[rospy.core][INFO] 2021-09-09 15:50:00,853: signal_shutdown [signal-2]
[rospy.internal][INFO] 2021-09-09 15:50:00,856: topic[/execution_check] removing connection to /server_endpoint
[rospy.internal][INFO] 2021-09-09 15:50:00,857: topic[/rosout] removing connection to /rosout
[rospy.internal][ERROR] 2021-09-09 15:50:00,857: Unable to initiate TCP/IP socket to wng:33025 (http://wng:34891/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 600, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2021-09-09 15:50:00,857: topic[/execution_check] removing connection to http://wng:34891/
[rospy.impl.masterslave][INFO] 2021-09-09 15:50:00,858: signal-2
[rospy.core][INFO] 2021-09-09 15:50:00,973: signal_shutdown [atexit]
